.github/workflows/check-licenses.yml

```diff
@@ -0,0 +1,30 @@
+name: Check Licenses
+on:
+  pull_request:
+    paths:
+      - 'requirements/requirements.txt'
+      - 'requirements/requirements.in'
+
+jobs:
+  license-check:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.9]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: install dependencies
+        run: |
+          python3 -m venv .venv-raiden
+          source .venv-raiden/bin/activate
+          make install
```

Comment:
This should still be faster than normal CI, but it might be nice to add some caching here.

Reply:
I don't know what kind of caching would make sense here.

The main point is, that this action triggers only when the requirements change anyways, so 
caching of the venv by hashing the requirements files would only make sense force successive runs for the same changes in requirements (e.g. triggered by force-push).

The other caching strategy could be to cache a "global" pip-cache (cache key not depending on requirement hashes) in order to speed up download time for wheels. Downsides: probably more cache-bloat, and the wheels still have to be installed.

In general I think the advantages are marginal compared to the integration test time.

Reply:
I agree, if it turns out to be a problem we can still fix it later.